CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Bob versions changelog
 - Added integration tests for aliens (#642)
 - Added integration tests for bobt (#648)
 - Added authentification support for integration tests (#672)
+- Added optional get & exist optimization (#702)
```

Comment:
Move this line to `[Unreleased]` section

Reply:
It is already there

Reply:
Moved after to [Unreleased] after resolving conflict